libraries/rush-lib/src/logic/buildCache/getHashesForGlobsAsync.ts

```diff
@@ -13,16 +13,16 @@ async function expandGlobPatternsAsync(
 ): Promise<string[]> {
   const allMatches: Set<string> = new Set<string>();
 
-  const { default: glob } = await import('glob');
-  const globAsync = (pattern: string, options: IOptions = {}): Promise<string[]> => {
-    return LegacyAdapters.convertCallbackToPromise(glob, pattern, options);
+  const { default: glob } = await import('fast-glob');
+  const globAsync = (pattern: string, options: Options = {}): Promise<string[]> => {
```

Comment:
Don't think this is needed anymore IMO

Reply:
I want to kill this function but haven't yet. That'll happen when all cache hashes get computed in advance of execution.